<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkWidget</class>
 <widget class="QMainWindow" name="WorkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ModelLab">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-50</y>
      <width>1117</width>
      <height>802</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="styleSheet">
    <string notr="true">background-color: #DAEFFF</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="GaugePanel" name="gaugepanel" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="GaugePanel" name="gaugepanel_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="GaugePanel" name="gaugepanel_3" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="styleSheet">
    <string notr="true">background-color: #DAEFFF</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="GaugePanel" name="gaugepanel_4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="GaugePanel" name="gaugepanel_5" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="GaugePanel" name="gaugepanel_6" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugePanel</class>
   <extends>QWidget</extends>
   <header>gaugepanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
